demo/README.md

```diff
@@ -102,7 +102,9 @@ Please send pull requests if you find ones that are missing here.
 - Gilberto Titericz, Stanislav Semenov, 1st place in challenge to classify products into the correct category organized by Otto Group in 2015. Link to [challenge](https://www.kaggle.com/c/otto-group-product-classification-challenge). Link to [kaggle winning solution](https://www.kaggle.com/c/otto-group-product-classification-challenge/discussion/14335)
 - Darius Barušauskas, 1st place winner in [Predicting Red Hat Business Value](https://www.kaggle.com/c/predicting-red-hat-business-value). Link to [interview](https://medium.com/kaggle-blog/red-hat-business-value-competition-1st-place-winners-interview-darius-baru%C5%A1auskas-646692a2841b). Link to [discussion](https://www.kaggle.com/c/predicting-red-hat-business-value/discussion/23786)
 - David Austin, Weimin Wang, 1st place winner in [Iceberg-classifier-challenge](https://www.kaggle.com/c/statoil-iceberg-classifier-challenge/leaderboard) Link to [discussion](https://www.kaggle.com/c/statoil-iceberg-classifier-challenge/discussion/48241)
-
+- Kazuki Onodera, Kazuki Fujikawa, 2nd place winner in [OpenVaccine: COVID-19 mRNA Vaccine Degradation Prediction](https://www.kaggle.com/c/stanford-covid-vaccine/overview) Link to [Discussion](https://www.kaggle.com/c/stanford-covid-vaccine/discussion/189709)
+- Artyom Kulakov, First player winner in [OSIC Pulmonary Fibrosis Progression](https://www.kaggle.com/c/osic-pulmonary-fibrosis-progression/overview) Link to [Discussion](https://www.kaggle.com/c/osic-pulmonary-fibrosis-progression/discussion/189346)
```

Comment:
I don't think xgboost is used here?

Reply:
They mentioned it in their report

Reply:
@Wittty-Panda No, their final report does not use XGBoost. They tried it but decided against using it. Quote:

> Tested other models, XGBoost, Log Regressions on tabular data. Thanks to my CV it immediately turned out that trees do not work here, so I didn't do anything with trees since the beginning of this competition.

Reply:
Okay I understand.
How can I make these changes ?

Reply:
@Wittty-Panda You should remove this item and replace with another winning solution that uses XGBoost.

Reply:
Should I make another commit or pull request ?
Sorry I'm new to this workflow

Reply:
Push a new commit will be fine. ;-)

Reply:
I committed as per you asked should I do anything else ?

Reply:
@Wittty-Panda You need to push your commit.

Reply:
@trivialfis @hcho3  I pushed it please accept 
sorry for the delay